src/store/modules/shipment/actions.ts

```diff
@@ -13,11 +13,11 @@ const actions: ActionTree<ShipmentState, RootState> = {
     try {
       resp = await ShipmentService.fetchShipments(payload)
       if (resp.status === 200 && resp.data.count> 0 && !hasError(resp)) {
-        let shipments = resp.data.docs;
-        if (payload.viewIndex && payload.viewIndex > 0) shipments = state.shipments.list.concat(shipments)
```

Comment:
Sir, as we have declared state.shipments.list as object in our state, and concat function does not work for objects.
Pagination is not implemented on this page because when we click on load more shipments upcoming shipments get added to the list of shipments that are on screen.

Reply:
Please check this code again, we are setting list
https://github.com/hotwax/receiving/blob/main/src/store/modules/shipment/actions.ts#L16
https://github.com/hotwax/receiving/blob/main/src/store/modules/shipment/mutations.ts#L7

It seems we are initializing with wrong value
https://github.com/hotwax/receiving/blob/main/src/store/modules/shipment/index.ts#L13

Fix this as well